ownCloud.xcodeproj/project.pbxproj

```diff
@@ -1486,6 +1489,7 @@
 				DCE5E8AD2080D781005F60CE /* folder-starred.tvg in Resources */,
 				DCE5E8A22080D781005F60CE /* folder-external.tvg in Resources */,
 				DC85572D20513B8C00189B9A /* ServerListTableViewController.xib in Resources */,
+				6E938522220322F30049D676 /* SplashImage.png in Resources */,
```

Comment:
Why do you not store the SplashImage.png in Assets.xcassets and why do you not provide @2x @3x resources?

Reply:
It's a 2048x2048 Image, so iOS reescalation works fine in this case.

We're not using assets catalog because of branding options. It's easier to exchange two images rather than make an asset catalog and exchange it with the actual one.

About the background color, we're used to switch the color modifying directly the *.XIB* file in the automatic-branded build process.

If you want me to use assets catalog I can do it but then we need to think how to create one in automatic-branded build process in our CI/build systems.

Reply:
Please keep the current behavior. It makes sense for branding. Maybe we will change it later.